code/modules/uplink/uplink_item.dm

```diff
@@ -299,6 +299,19 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	item = /obj/item/melee/powerfist
 	cost = 8
 
+/datum/uplink_item/dangerous/mindbatterer
+	name = "Psychic Disruptor"
+	desc = "A new Epsilon Psi Corps Disruptor Device, also known as Mind Batterer. Creates an invisible wave \
+		of psychic energy that causes immense pain to people near the user \
+		which may force them to the ground, effectivly disabling them for some time. \
+		The device emits a very loud buzz everyone hears in their head. \
+		It has 3 uses, mass produced version of The Epsilon Army." 
```

Comment:
`The device emits a very loud buzz everyone hears in their head.` is really redundant and should be scrapped.

`The Epsilon Army` is not anything that we have in our lore and it sounds more edgy than anything else. I'd also scrap that.

Reply:
Was a subtle reference to C&C Red Alert 2 from which said device (the original batterer that is, spritewise and functional) was taken from.